packages/docs/src/pages/tutorial/events/basic/index.mdx

```diff
@@ -29,17 +34,17 @@ To better understand what is going on we have included simplified HTML of the ap
 </html>
 ```
 
-
-1. The APIs contain `$` to ensure that your code contains them.
-2. The Optimizer looks for `$` and extracts the function wrapped by `$` into a separate lazy-loadable chunk. (See the documentation to better understand how this is done.)
-3. As part of the SSR the server executes the JSX and notices that there is a click listener. The click listener gets serialized into the `<button>` element as `on:click` attribute. (The unique behavior is that the Qwik knows how to subscribe to events as part of the SSR.)
-4. Qwikloader scripts get inlined into HTML. The Qwikloader script sets up a global listener for all events in the browser. (Qwikloader is about 1kb and executes in about 1ms.)
-5. When a user clicks on the button the Qwikloader intercepts the event and looks for an element with `on:click` attribute.
+1. The APIs contain the `$` to ensure that your code contains them.
```

Comment:
```suggestion
1. APIs contain the `$` so the Qwik optimizer can transform your code for lazy-loading.
  - The `$` is a signals to that lazy-loading "magic" happens at this point.
  - TypeScript rules ensure you don't skip any places where the `$` should appear.
```

@mhevery - how's that?

Reply:
❤️  it

Reply:
I would also add 
```
- The `$` is a signal to Optimizer to extract the code for lazy-loading
```
to the list at the first position.